### ISSUE TITLE
Improvement: Set the default Desktop Browser Dimensions to 1920x1080 as its the most current most popular dimension in the world

### ISSUE THREAD

**Issue by d19dotca** [WPO-Foundation/wptagent#408]
I think it may be time to change the default Desktop Browser Dimensions setting to a resolution of 1920x1080 from the current default of 1366x768.

In all of my own web analytics, the 1920x1080 is the most common desktop browser dimension, and upon investigating this further it is also confirmed by third-parties such as StatCounter: https://gs.statcounter.com/screen-resolution-stats

Worldwide the current stats are in the image below:
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/8153796/113495787-89aa7b80-94a8-11eb-9b16-29efa2bacb42.png">

In specific regions, the difference between the two resolutions are even greater: 
<img width="1210" alt="image" src="https://user-images.githubusercontent.com/8153796/113495824-f02f9980-94a8-11eb-84e2-5ad3f5f5a769.png">
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/8153796/113495830-00477900-94a9-11eb-8ecd-efd39f5e8ec8.png">
<img width="1208" alt="image" src="https://user-images.githubusercontent.com/8153796/113495916-a2fff780-94a9-11eb-86ed-4df3805733b2.png">

The two regions where the 1366x768 is more popular is in the Asia and South America regions:
<img width="1209" alt="image" src="https://user-images.githubusercontent.com/8153796/113495847-1e14de00-94a9-11eb-990f-a0fe108b4d95.png">
<img width="1217" alt="image" src="https://user-images.githubusercontent.com/8153796/113495937-bc08a880-94a9-11eb-9146-2e40d5742f26.png">

But since this website is used globally, I believe we should pay attention to the overall worldwide stats. Across the world on average the 1920x1080 browser dimension is the most popular one. So I'd like to propose that the default browser dimension setting be changed from the current default 1366x768 to the more popular 1920x1080.

**Comment by d19dotca** [comment]
I just realized I submitted this to the wrong project. This was meant for the WebPageTest project.

**Comment by tkadlec** [comment]
Thanks @d19dotca! We made the change to 1366x768 super recently (March 23rd). We did consider the 1920x1080 resolution for a default, as both are _super_ close if you look at desktop only traffic: https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide

In the end, we went with the smaller resolution as it's what you find on the cheaper laptops and Chromebooks, as opposed to the higher resolution being found on more powerful machines (typically).

We'll definitely keep an eye on it though.
